tests/test_app.py

```diff
@@ -102,6 +264,10 @@ class TestFirebaseApp(object):
         firebase_admin.App('uninitialized', CREDENTIAL, {})
     ]
 
+    bad_config_file = ['firebase_config_empty.json',
```

Comment:
Inline this list wherever you're using it. Seems to be used in only one test case.